# This contains the full state of the space vehicle
float32 phi       # Roll, angle rotated around i^(vehicle-2)
float32 theta     # Pitch, angle rotated around j^(vehicle-1)
float32 psi       # Yaw, angle rotated around k^(vehicle)
float32 p         # Roll rate around i^(body)
float32 q         # Pitch rate around j^(body)
float32 r         # Yaw rate around k^(body)
